spark-jobs/src/main/scala/filodb/downsampler/index/DSIndexJobSettings.scala

```diff
@@ -28,6 +28,9 @@ object DSIndexJobSettings extends StrictLogging {
 
   val cassWriteTimeout = dsIndexJobConfig.as[FiniteDuration]("cassandra-write-timeout")
 
+  //default 6hours
+  val batchLookbackInHours = dsIndexJobConfig.as[Option[Int]]("batch-lookback-in-hours").getOrElse(6)
```

Comment:
Pls document clearly why we configure lookback time instead of rerun only the period that failed.
We need to highlight that updates to entries applied at later time could be lost if replay is not done.

Reply:
ack, fixed it